src/main/java/appeng/client/render/cablebus/QuadRotator.java

```diff
@@ -35,6 +38,20 @@
  */
 public class QuadRotator implements RenderContext.QuadTransform {
 
+    /**
+     * See {@link AoCalculator} for the source of this data.
+     */
+    private static final int[][] VERTEX_MAP = new int[6][4];
```

Comment:
Is this used somewhere?

Reply:
Oh, true, no. I was first trying to explicitly swivel the vertices after rotation but honestly was too lazy to actually figure out the correct swivel :D